src/control/server/init/setup_spdk.sh

```diff
@@ -19,11 +19,12 @@ else
 
 	# build arglist manually to filter missing directories/files
 	# so we don't error on non-existent entities
-	for glob in /dev/hugepages '/dev/uio*'		\
+	for glob in '/dev/hugepages' '/dev/uio*'		\
 		'/sys/class/uio/uio*/device/config'	\
 		'/sys/class/uio/uio*/device/resource*'; do
 
-		if list=$(ls "$glob"); then
+		if list=$(ls $glob); then
+			echo "RUN: ls $glob | xargs -r chown -R $_TARGET_USER"
```

Comment:
I realise that this is just an info print but would it be more accurately expressed as:

```suggestion
			echo "RUN: echo $glob | xargs -r chown -R $_TARGET_USER"
```

to reflect what is actually being done below?

Reply:
I will make this change if I have to re-push/tests fail